ui/lib/components/CardTableV2/index.tsx

```diff
@@ -122,7 +122,7 @@ function CardTableV2(props: ICardTableV2Props) {
     >
       <div className={styles.cardTableContent}>
         <ShimmeredDetailsList
-          selectionMode={SelectionMode.none}
+          selectionMode={selectionMode ?? SelectionMode.none}
```

Comment:
I guess the `...restProps` can override this prop, so maybe we don't need to change this part logic, an example in ui/lib/apps/Statement/pages/Detail/index.tsx:

```js
              <CardTableV2
                cardNoMargin
                columns={planColumns}
                items={plans}
                selectionMode={SelectionMode.multiple}
                selection={selection.current}
                selectionPreservedOnEmptyClick
              />
```